src/__tests__/postgres/integration.test.ts

```diff
@@ -394,4 +397,55 @@ describeMaybeSkip('integration (postgres)', () => {
     expect(errors).toBeUndefined()
     expect(data?.films.some((f: any) => f.sequel && f.sequel.id)).toBe(true)
   })
+
+  test('subscriptions', async () => {
+    const document = parse(`
+      subscription {
+        create 
+      }
+    `)
+
+    const sub = <AsyncIterator<ExecutionResult>>await subscribe(schema, document);
+    expect(sub.next).toBeDefined()
+    setImmediate(() => pubsub.publish('CREATE_ONE', { create: "FLUM!" }))
```

Comment:
issue: we should be testing this using the `publish` method on the client you added

The difference in coverage can be seen [here](https://coveralls.io/builds/31519894/source?filename=src%2FqueryBuilder%2FcreateOne.ts#L17) -- you can get to the coveralls panel by clicking on the `coverage` badge in the comment from the coveralls bot.

There are three new uncovered lines from the `publish` method because we never test it anywhere. Before we do integration tests, it might make sense to just add unit tests for the `publish` method itself -- both for when a `PubSub` instance is provided and when it's not to cover both branches. Adding integration tests for subscriptions can be something we do in a separate PR since it will take considerable work -- as long as we have unit tests for any added functionality, I think we're ok.

Reply:
There's a test at the end of integration.js that attempts to do this, currently skipped because it times out.  It's using mockResolveInfo to do the mutation, but I see now there is no client being passed to it.  I'll look at this more.

Reply:
That test appears to be working now.